README.md

```diff
@@ -1,76 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Catalog of my things
 
-> One paragraph statement about the project.
+> This app helps users to keep a record of different types of things they own
 
-![screenshot](./app_screenshot.png)
+This is a simple console app that allows user to manage collections of books they own
 
-Additional description about the project and its features.
+This app can do the following:
 
-## Built With
+- Show a list of books
+- Show a list of labels
+- Store books
 
-- Major languages
-- Frameworks
-- Technologies used
+All the books and labels are stored in JSON files
```

Comment:
- [ ] [OPTIONAL] it might be nice if you added a .gif to your readme file in order to showcase your application, I think that would make it look a bit more professional as well.